OutOfSchool/OutOfSchool.WebApi/Services/Database/WorkshopService.cs

```diff
@@ -92,7 +92,7 @@ public async Task<WorkshopBaseDto> Create(WorkshopBaseDto dto)
 
         if (dto.AvailableSeats is 0 or null)
         {
-            dto.AvailableSeats = uint.MaxValue;
+            workshop.AvailableSeats = uint.MaxValue;
```

Comment:
(Tests) Test coverage is 100%, but I don't see any test changed/failed while logic was changed. Could you check and write missing test if necessary for this case?

Reply:
I've fixed tests, but I'm not sure is it ok to add same guid in every method mock or should I do same as with avaliable seats?

Reply:
1. (Readability) If you really need same `GUID` - please introduce it as local variable before and use it in setup, don't recreate same each time. This will make reuse of same `id` more obvious (without keeping an eagle eye for few lines comparing 20+ symbols) 
2. (The logic) Honestly I don't get if `id` should be same - is any test failing because of wrong `id`?